core/station/impl/src/factories/requests/monitor_external_canister.rs

```diff
@@ -0,0 +1,88 @@
+use super::{Create, Execute, RequestExecuteStage};
+use crate::models::{MonitorExternalCanisterOperation, MonitorExternalCanisterOperationKind};
+use crate::{
+    errors::{RequestError, RequestExecuteError},
+    models::{Request, RequestExecutionPlan, RequestOperation},
+    services::ExternalCanisterService,
+};
+use async_trait::async_trait;
+use orbit_essentials::types::UUID;
+use std::sync::Arc;
+
+pub struct MonitorExternalCanisterRequestCreate;
+
+#[async_trait]
+impl Create<station_api::MonitorExternalCanisterOperationInput>
+    for MonitorExternalCanisterRequestCreate
+{
+    async fn create(
+        &self,
+        request_id: UUID,
+        requested_by_user: UUID,
+        input: station_api::CreateRequestInput,
+        operation_input: station_api::MonitorExternalCanisterOperationInput,
+    ) -> Result<Request, RequestError> {
+        let request = Request::new(
```

Comment:
this should now align with the latest format in the main branch

Reply:
yes, this is covered in https://github.com/dfinity/orbit/pull/419/commits/26b8669251c1162e70da964e401ea49507f589ad